### DISCUSSION TITLE
Got Ender3V2S1 firmware installed on ENDER 5 PRO- 4.2.7 board with blt and new V2 display working, just 1 problem... Homing is in wrong direction

### DISCUSSION THREAD

**Discussion by LambertCFI** [mriscoc/Ender3V2S1#1049] [Q&A]
How do I get My Ender 5 Pro to home in the back right corner?  I have the Ender3V2S1 firmware installed and the V2 display working I cant figure out how to rest my home position on the Ender 5 back right corner.  Unlike the Ender 3 it homes in the front left corner as it should be.  It seems the Ender 3 and Ender 5 are pretty much the same with a 4.2.7 board except for where they home position is.

Also I havent seen BLTouch any where on the different pages in on the display. any help ther would be great as well thanks in advance!

Scott

**Comment by mriscoc** [comment]
You could move the endstops, or build a custom version of the firmware.
